website/src/pages/index.md

```diff
@@ -1,11 +1,5 @@
-# Component Library
+# Europa Component Library
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut convallis
-metus non lectus dignissim, ac viverra elit convallis. Pellentesque
-tempus purus id facilisis lacinia.
+The **Europa Component Library** (ECL) is a living style guide containing the design standards and visual assets for the European Commission websites hosted under the ec.europa.eu domain and the European Union website hosted under the europa.eu domain.
```

Comment:
`Commission websites` or  `Commission's websites`?

Reply:
Our content writer wrote it Commission websites. So it is correct.